README.md

```diff
@@ -10,7 +10,7 @@
 [quay]: https://quay.io/repository/cyberark/conjur "Conjur container image on Quay.io"
 [twitter]: https://twitter.com/intent/user?screen_name=ConjurInc "Follow Conjur on Twitter"
 
-Conjur provides secrets management and machine identity for modern infrastructure:
+Conjur provides secrets management and application identity for modern infrastructure:
```

Comment:
changed machine -> application identity b/c machine identity will no longer be part of the host name?

Reply:
we renamed `machine identity` to `application identity` altogether